File: Q13/Q13.ts
// Count the number of vowels in a string.

function countVowelsTS(str: string): number {
  const vowels = "aeiouAEIOU";
  let count = 0;
  for (let char of str) {
    if (vowels.includes(char)) {
      count++;
    }
  }
  return count;
}

// Example usage
const inputStrTS = "Hello World!";
console.log(countVowelsTS(inputStrTS)); // Output: 3

File: Q13/Q13.py
# Count the number of vowels in a string.

def count_vowels(s: str) -> int:
    vowels = "aeiouAEIOU"
    count = 0
    for char in s:
        if char in vowels:
            count += 1
    return count

input_str= "Hello World!"
print(count_vowels(input_str))
